packages/loot-core/src/client/query-helpers.ts

```diff
@@ -234,7 +236,7 @@ class PagedQuery extends LiveQuery {
 
       const orderDesc = getPrimaryOrderBy(this.query, defaultOrderBy);
       if (orderDesc == null) {
-        throw new Error(`refetchUpToRow requires a query with orderBy`);
+        throw new Error(t('refetchUpToRow requires a query with orderBy'));
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏do we actually show this error in the UI? If not - I don't think we need a translation for it.

Reply:
From grepping through the codebase, this is actually dead code 😅 I'll remove translations for this, but do you think it's worth deleting? I know very little about this part of the code

Reply:
If it's not used then yea - it makes sense to remove it. :)